client/components/Login/index.jsx

```diff
@@ -72,13 +72,29 @@ class Login extends Component {
             }
         )).json();
 
+        const { permissions } = await (await fetch(
+            '/api/roles/permission'
+        )).json();
+
         if (error) {
             this.setState({ loginError });
             this.props.logOut('');
         } else {
-            Session.create({ username, timeout: Date.now() });
+            this.props.logIn({
+                username,
+                user_id: id,
```

Comment:
Why does this need to be renamed? What other data would `logIn` accept that might be ambiguous?

Reply:
`logIn` doesn't accept other data, but I thought the number of `id`s floating around might get confusing (ie scenario ids, cohort ids). If you don't think it's confusing here, I can change it.

Reply:
Oh actually this is leftover from another approach, I don't think we need it after all.